host/sgx/windows/enter.asm

```diff
@@ -73,6 +74,11 @@ NESTED_ENTRY oe_enter, _TEXT$00
     mov rax, [rbp+64]
     mov ENCLAVE, rax
 
+    ;;Save the current context
+
+    ;;Save the SSE status and control flags
```

Comment:
This could use some more explanation (considering you had a whole presentation explaining what went on here 😄 )